includes/amp-helper-functions.php

```diff
@@ -1983,3 +1947,39 @@ function amp_generate_script_hash( $script ) {
 	);
 	return 'sha384-' . $hash;
 }
+
+/**
+ * Get the AMP version of a URL.
+ *
+ * @since 2.1
+ * @todo Rename this to clear up confusion with amp_get_current_url().
+ *
+ * @param string $url URL.
+ * @return string AMP URL.
+ */
+function amp_get_url( $url ) {
```

Comment:
We have `amp_get_current_url()` and `amp_get_asset_url()` and `amp_get_customizer_url()`. These being here, I think `amp_get_url()` is too ambiguous. 

How about we call it `amp_get_paired_url()` since its purpose is to get the paired AMP URL for the current page. 

We also have a `\AMP_Theme_Support::get_current_canonical_url()` which could be extracted into a separate global function called `amp_get_canonical_url()`.

Reply:
Done in b66c233fa